### PR TITLE
MMB insurance: design order summary overview

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

File: app/MMB/src/scenes/tripServices/insurance/OrderSummary.js [hunk 1]
```diff
@@ -2,36 +2,41 @@
...
-      <Price>
-        <Translation passThrough="TODO" />
-      </Price>
+      <View style={styleSheet.row}>
+        <View style={styleSheet.item}>
+          <Text>
+            <Translation passThrough="TODO" />
+          </Text>
+        </View>
+        <Price amount={-1} currency="WTF" style={styleSheet.price} />
```
Comment (RobinCsl) [inline app/MMB/src/scenes/tripServices/insurance/OrderSummary.js]: Nice currency code :)

Comment (tbergq) [inline app/MMB/src/scenes/tripServices/insurance/OrderSummary.js]: WTF? 😄 WIP?

Comment (mrtnzlml) [reply app/MMB/src/scenes/tripServices/insurance/OrderSummary.js]: yes

Comment (mrtnzlml) [reply app/MMB/src/scenes/tripServices/insurance/OrderSummary.js]: wip :)

File: packages/shared/src/Price.js [hunk 2]
```diff
@@ -8,9 +8,9 @@ import Text from './Text';
 import type { StylePropType } from '../types/Styles';
 
 type Props = {|
-  amount: number | null, // may be null (compatible with GraphQL query failure)
-  currency: string | null, // may be null (compatible with GraphQL query failure)
-  style?: StylePropType,
+  +amount: number | null, // may be null (compatible with GraphQL query failure)
```
[the remark this answers is not in the document]
Comment (mrtnzlml) [reply packages/shared/src/Price.js]: Removed. It was just an explanation because it's not really clear from this component (it's GraphQL agnostic)...

**Pull request merged**